plugins/MauticCrmBundle/Entity/PipedriveDeletion.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace  MauticPlugin\MauticCrmBundle\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
+
+class PipedriveDeletion
+{
+    /**
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $objectType;
+
+    /**
+     * @var int
+     */
+    private $integrationEntityId;
+
+    /**
+     * @var int
+     */
+    private $deletedDate;
+
+    public static function loadMetadata(ORM\ClassMetadata $metadata)
+    {
+        $builder = new ClassMetadataBuilder($metadata);
+        $builder->setTable('plugin_crm_pipedrive_deletions');
+
+        $builder->addId();
+        $builder->addNamedField('objectType', 'string', 'object_type');
```

Comment:
Let's use constants whenever possible. Like for column types:
```suggestion
        $builder->addNamedField('objectType', Types::STRING, 'object_type');
```